suite/tests/client-interface/drreg-test.c

```diff
@@ -67,6 +67,18 @@ void
 test_asm_faultK();
 void
 test_asm_faultL();
+void
+test_asm_faultM();
```

Comment:
Should these have descriptive names rather than just letters, with so many?

Reply:
Done.